rel/emqx_vars

```diff
@@ -22,5 +22,6 @@ export EMQX_DESCRIPTION='{{ emqx_description }}'
 ## computed vars
 REL_NAME="emqx"
 ERTS_PATH="$RUNNER_ROOT_DIR/erts-$ERTS_VSN/bin"
+RUNNER_DATA_DIR="${EMQX_NODE__DATA_DIR:-$RUNNER_DATA_DIR}"
```

Comment:
This will change not only the mnesia dir, but also the [configs dir](https://github.com/emqx/emqx/blob/8d26ba6e9f48d6a39148e5a66d63ca257e05d00d/bin/emqx#L35), right?